packages/common/src/blobStorage.js

```diff
@@ -0,0 +1,27 @@
+const { BlobServiceClient } = require('@azure/storage-blob');
+const logger = require('./lib/logger');
+
+const initContainerClient = async (config) => {
```

Comment:
This was originally in document-service-api, moved it here as we wanted it in appeals api